index.html

```diff
@@ -10,6 +10,7 @@
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
 
 	</head>
 	<body>
```

Comment:
You should NEVER have more than one body tag in your HTML document. Your JS file should have been linked right under the query.min.js.